src/Shopsys/ShopBundle/DataFixtures/Demo/CategoryDataFixture.php

```diff
@@ -71,6 +71,7 @@ public function load(ObjectManager $manager)
         $categoryData->name = [
             'cs' => 'Elektro',
             'en' => 'Electronics',
+            'de' => '(DE) Electronics',
```

Comment:
@LukasHeinz I'm not sure whether this demonstration of translation is enough. Are we planning to fix German demo data sometime in the future?

![image](https://user-images.githubusercontent.com/10008612/63945547-c9f77b80-ca73-11e9-9049-b889c1155b26.png)

Reply:
All translatable messages were just appended with `(DE)` (except countries which were actually translated)